--- Xnee-3.01/cnee/src/cnee_printer.c.orig	2007-06-28 12:16:09.000000000 +0200
+++ Xnee-3.01/cnee/src/cnee_printer.c	2007-07-14 21:01:06.369193162 +0200
@@ -400,7 +400,10 @@
   fprintf (fd ,"@setchapternewpage off\n");
   fprintf (fd ,"@c %%**end o\n");
 
-
+  fprintf (fd, "@dircategory Miscellaneous:\n");
+  fprintf (fd, "@direntry\n");
+  fprintf (fd, "* Xnee: (cnee).\t\t\t\tXnee's Not Event Emulator\n");
+  fprintf (fd, "@end direntry\n");
 
   fprintf (fd, "@ifinfo\n");
   fprintf (fd, "@node Top, Options,, (dir)\n");
